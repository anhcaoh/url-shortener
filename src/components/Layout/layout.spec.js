import React from "react";
import renderer from "react-test-renderer";
import { Row, Columns, Column, Card, Flexbox } from "./index";
 
describe("Layout", () => {
    it("should render snapshot", () => {
            const component = renderer.create(<Row>
                <Columns>
                    <Column>
                        <Flexbox>
                            <Card>Content inside layout components</Card>
                        </Flexbox>
                    </Column>
                </Columns>
                </Row>);
            let tree = component.toJSON();
            expect(tree).toMatchSnapshot(); 
    });
});