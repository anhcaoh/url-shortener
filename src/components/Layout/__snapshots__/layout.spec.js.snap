// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render snapshot 1`] = `
<div
  className="row"
>
  <div
    className="columns"
  >
    <div
      className="column"
    >
      <div
        className="flexbox"
      >
        <div>
          <div
            className="card"
          >
            Content inside layout components
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
